ext/curl/tests/curl_postfields_array.phpt

```diff
@@ -0,0 +1,63 @@
+--TEST--
+CURLOPT_POSTFIELDS with multi-value fields
+--EXTENSIONS--
+sockets
+--FILE--
+<?php
+$socket = stream_socket_server("tcp://0.0.0.0:29999", $errno, $errstr);
```

Comment:
Any particular reason why you don't use the test server (see server.inc)?

Reply:
Yes, because the PHP server's HTTP implementation parses the raw POST data into an array (and you can't get raw POST data with `multipart/form-data` through `php://input`). The parsed array then has the problem of overriding the duplicate key, making the test useless. (I tried that first)